docs/manuals/core/config/job-config.md

```diff
@@ -114,4 +114,71 @@ For example, the following configuration will allow Taipy to run up to eight `Jo
     max_nb_of_workers = "8:int"
     ```
 
+# Cluster mode
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+The *cluster* mode is designed to make a Taipy application executes jobs on a remote cluster of
+distributed workers. The number of machines available in the cluster is unlimited, so the
+application becomes fully scalable horizontally. It can also be adjusted dynamically depending on
+some load metric.
+
+With the *cluster* mode, a `Job^` asynchronously runs in its own environment. At the submission,
+the job is queued in a RabbitMQ queue. When there is a worker available, the job is dequeued and
+sent to the worker environment to be executed.
+
+Note that with the *cluster* mode, the submit method is not blocking and returns after the job is queued.
+It means the submit method can return before the job finishes or even before it is dequeued. Please refer to
+the [submit entity](../entities/orchestrating-and-job-execution.md#submit-a-scenario-sequence-or-task) section
+on how to submit jobs.
+
+You can configure the *cluster* mode with the following config:
+
+=== "Python configuration"
+
+    ```python linenums="1"
+    from taipy import Config
+
+    Config.configure_job_executions(mode="cluster")
+    ```
+
+=== "TOML configuration"
+
+    ```python linenums="1"
+    from taipy import Config
+
+    Config.load("config.toml")
+    ```
+
+    ```toml linenums="1" title="config.toml"
+    [JOB]
+    mode = "cluster"
+    ```
+
+To set up a worker environment, some requirements must be met:
+
+- The worker environment must have access to the same Taipy application as the main environment.
+- The worker environment must have the same Python environment as the main environment.
+- The worker environment must have access to the same RabbitMQ server as the main environment.
```

Comment:
Do we need to expose the techno here?

Reply:
Most is necessary I think. However, for the "RabbitMQ", it may be too specific. What do you think @jrobinAV?

Reply:
That is a prerequisite. We rely on RabbitMQ for this feature. I would keep it here.